docs/source/torch.compiler_cudagraph_trees.rst

```diff
@@ -90,8 +93,9 @@ The second time we hit graph 3 we are warmed up and ready to record. We record g
    \\   \\
     4   4
 
-Limitations
------------
+
+Iteration Support
```

Comment:
nit: i'm not sure `IterationSupport` means anything to people who are not previously familiar. maybe just keep as Limitations